templates/nglayouts/themes/standard/layout/layout_1.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends nglayouts.pageLayoutTemplate %}
+
+{% block layout %}
+    <div class="zone-layout-layout1">
+
+        <main class="zone zone-main">
```

Comment:
we need <main class="main-content-block"> here as well because we will link this in Skip to main content - you can add link there within this PR

Reply:
added "<main class="main-content-block">"